x-pack/platform/plugins/shared/cases/common/types/api/case/v1.ts

```diff
@@ -533,12 +533,12 @@ export const GetRelatedCasesByAlertResponseRt = rt.array(RelatedCaseRt);
 
 export const SimilarCasesSearchRequestRt = paginationSchema({ maxPerPage: MAX_CASES_PER_PAGE });
 
-export const FindCasesContainingAllAlertsRequestRt = rt.exact(
+export const FindCasesContainingAllDocumentsRequestRt = rt.exact(
   rt.type({
     /**
-     * The IDs of the alerts to find cases for.
+     * The IDs of the documents to find cases for.
      */
-    alertIds: rt.array(rt.string),
+    documentIds: rt.array(rt.string),
```

Comment:
Changes like this would need to be rolled out with an intermediate release.

Reply:
even if the api is internal?